client/src/tocs-event-handler.ts

```diff
@@ -204,10 +208,18 @@ export class TocsEventHandler implements vscode.TreeDragAndDropController<BookOr
 
   async handleDrop(target: BookOrTocNode | undefined, dataTransfer: vscode.DataTransfer): Promise<void> {
     const dragging: BookOrTocNode | undefined = dataTransfer.get(XFER_ITEM_ID)?.value
-    if (dragging?.type === undefined) throw new Error('BUG: Bad drag target')
-    if (target === undefined) throw new Error('BUG: Bad drop target')
-    if (target !== dragging && dragging.type !== BookRootNode.Singleton) {
-      await this.moveNode(dragging, target)
+    if (dragging?.type === undefined) { throw new Error('BUG: Bad drag target') }
+    if (target === undefined) { throw new Error('BUG: Bad drop target') }
+    if (
+      target !== dragging &&
+      dragging.type !== ClientOnlyTocKinds.OrphanCollection &&
+      dragging.type !== BookRootNode.Singleton
+    ) {
+      if (target.type === ClientOnlyTocKinds.OrphanCollection) {
+        await this.removeNode(dragging)
```

Comment:
So they never get put in the OrphanCollection, they just go under it somehow? How does the OrphanCollection actually gets elements, like in your screenshot?

Reply:
- Orphans are sent by the language server [here](https://github.com/openstax/poet/blob/30dcb65f7a3449418ef89e3aeb3e7141065c2d56/server/src/model-manager.ts#L173) after the remove event is handled.
- [This](https://github.com/openstax/poet/blob/de8a57e6e4827904686999c21a9767cf945d8db2/client/src/extension.ts#L102) updates the client's toc tree with books and orphans.
- Orphans are added to the OrphanCollection [here](https://github.com/openstax/poet/blob/4bdf7c59fef017cf3bfa929778b2b5d32f1537a1/client/src/book-tocs.ts#L47) on update.

Reply:
Also, here's a video instead of a screenshot. 

https://github.com/user-attachments/assets/17d6f173-3b70-4b39-8a85-a80353e2ddb7